src/main/java/com/github/leroyguillaume/keycloak/bcrypt/BCryptPasswordHashProvider.java

```diff
@@ -54,12 +59,22 @@ public void close() {
 
     @Override
     public boolean verify(String rawPassword, PasswordCredentialModel credential) {
-        return BCrypt.checkpw(rawPassword, credential.getPasswordSecretData().getValue());
+        final String hash = credential.getPasswordSecretData().getValue();
+        BCrypt.Result verifier = BCrypt.verifyer().verify(rawPassword.toCharArray(), hash.toCharArray());
+        return verifier.verified;
     }
 
     private String generateBCryptSalt(int iterations) {
```

Comment:
You can delete this function because it is not used anymore

Reply:
Thx, removed